php/src/Google/Protobuf/Internal/GPBUtil.php

```diff
@@ -597,4 +609,65 @@ public static function hasJsonValue($msg)
                is_a($msg, "Google\Protobuf\StringValue") ||
                is_a($msg, "Google\Protobuf\BytesValue");
     }
+
+
+    private static $wrapperTypes = [
+        DoubleValue::class,
+        FloatValue::class,
+        Int64Value::class,
+        UInt64Value::class,
+        Int32Value::class,
+        UInt32Value::class,
+        BoolValue::class,
+        StringValue::class,
+        BytesValue::class,
+    ];
+
+    /**
+     * Determine if a fully qualified class name is a wrapper type.
+     *
+     * @param string $class the fully qualified class name
+     * @return bool true if this is a wrapper type
+     */
+    public static function isWrapperType($class)
```

Comment:
@TeBoring I would like to get your take on this method vs the existing `hasJsonValue` method. They check all the same classes, but have slightly different inputs (instance vs class name). I can reconcile into a single method, but wanted to get your input first on what you would prefer.